content/competitions/hackathon-2021.html

```diff
@@ -0,0 +1,122 @@
+---
+title: "Hackathon"
+draft: false
+---
+
+<div class="section columns is-centered">
+    <section class="column is-6">
+        
+        <div class="block">
+            <a href="https://www.facebook.com/events/2858423371101933/">
+                <img src="{{< baseurl >}}/img/media/competitions/hackathon_2021_banner.png" style="width: 100%" />
+            </a>
+        </div>
+
+        <h1 class="title is-3">UQCS Hackathon 2021</h1>
+        <p class="subtitle is-4">Friday 20th August &ndash; Sunday 22th August</p>
+        <p class="subtitle is-6">Registrations open at 5pm.</p>
+
+        <div class="content">
+            <div class="buttons has-text-centered">
+                <a href="https://events.humanitix.com/hackathon2021" class="button is-link uqcs-blue">Get your tickets now!</a>
+                <a href="https://www.facebook.com/events/200547242027862" class="button">Facebook event</a>
+            </div>
+
+			<p>
+				The much awaited UQCS Hackathon is back & better than ever! Taking place from the 20th to 22nd of August, this is a great time to learn a new framework, develop your programming and/or design skills, and win some money. Use the link above to buy tickets - make sure to buy them before <date> to get a LIMITED EDITION Hackathon 2021 t-shirt for ONLY $5 and early bird tickets for only $10!
```

Comment:
`<date>` is now 6th August.

Reply:
I have now fixed <date> 3 different times on 3 different platforms rip